tests/foreman/ui_airgun/test_ldap_authentication.py

```diff
@@ -763,3 +770,88 @@ def test_positive_add_katello_role_with_org(
     ak = entities.ActivationKey(organization=module_org).search(
         query={'search': 'name={}'.format(ak_name)})[0].read()
     assert ak.organization.id == module_org.id
+
+
+@skip_if_not_set('ldap')
+@tier2
+@upgrade
+def test_positive_create_user_in_ldap_mode(session, auth_source):
+    """Create User in ldap mode
+
+    :id: 0668b2ca-831e-4568-94fb-80e45dd7d001
+
+    :expectedresults: User is created without specifying the password
+
+    :CaseLevel: Integration
+    """
+    auth_source_name = 'LDAP-' + auth_source.name
+    name = gen_string('alpha')
+    with session:
+        session.user.create({
+            'user.login': name,
+            'user.auth': auth_source_name,
+        })
+        assert session.user.search(name)[0]['Username'] == name
+        user_values = session.user.read(name)
+        assert user_values['user']['auth'] == auth_source_name
+
+
+@skip_if_not_set('ldap')
+@tier2
+def test_positive_login_ad_user_no_roles(test_name, ldap_data, ldap_user_name, auth_source):
+    """Login with LDAP Auth- AD for user with no roles/rights
+
+    :id: 7dc8d9a7-ff08-4d8e-a842-d370ffd69741
+
+    :setup: assure properly functioning AD server for authentication
+
+    :steps: Login to server with an AD user.
+
+    :expectedresults: Log in to foreman UI successfully but cannot access
+        functional areas of UI
+
+    :CaseLevel: Integration
+    """
+    with Session(
+            test_name,
+            ldap_data['ldap_user_name'],
+            ldap_data['ldap_user_passwd'],
+    ) as ldapsession:
+        with raises(NavigationTriesExceeded):
+            ldapsession.user.search('')
```

Comment:
can we try to read current user somehow, to ensure expected result `Log in to foreman UI successfully`?